fileConventions/Library.fs

```diff
@@ -0,0 +1,9 @@
+﻿module fileConventions
+open System.IO
+open System
+
+let HasShebang (fileInfo: FileInfo) =
+    use streamReader = new StreamReader (fileInfo.FullName)
+    let fileText = streamReader.ReadToEnd()

```

Comment:
this can be simplified to use File.ReadAllText()

Reply:
Done